muikku-core-plugins/src/main/resources/fi/otavanopisto/muikku/plugins/evaluation/EvaluationJsPluginMessages_fi.properties

```diff
@@ -145,6 +151,13 @@ plugin.evaluation.evaluationModal.events.improvedGradeButton = Anna korotus
 plugin.evaluation.evaluationModal.events.supplementationButton = Pyyd� t�ydennyst�
 plugin.evaluation.evaluationModal.events.selectSubject = Valitse arvioitava moduuli
 
+plugin.evaluation.evaluationModal.journalComments.formContentLabel = Sis�lt�
```

Comment:
Missing or mismatched locales vs en locale?

Reply:
Should be fixed now